packages/fewer/src/Repository/index.ts

```diff
@@ -495,6 +544,8 @@ export function createRepository<Type extends SchemaTable>(
 ): Repository<
   Type[INTERNAL_TYPES.INTERNAL_TYPE],
   {},
+  Type['database']['adapter']['FunctionsType'],
```

Comment:
We should probably eat this on the schema table so that we don't need to deep-read here.